tests/base.py

```diff
@@ -1,4 +1 @@
-try:
```

Comment:
We can eliminate this file now and directly import TestCase in the test files.

Reply:
As a consequence, I re-ordered imports in `tests` folder. Not sure if you have a strong opinion on imports, but anyway I'm okay to change them if needed.